content/md/products-super-power/enrich-products-variants.md

```diff
@@ -266,6 +266,50 @@ The list of versions are displayed in the `History` tab.
 In `COMMON`, only the common properties (attributes and categories) will be versioned. You can use the navigation component to display the versions of each variant level.
 :::
 
+# Convert a variant product to a simple one
+
+Sometimes, you may need to convert a variant product to a simple product, for instance when there is a reduction of your product range.
+To do so, several choices:
+- directly in a variant product sheet
+- via a bulk action
+- via a product import
+- via the API
+
+Let's see how each of these solutions works!
+
+## Convert a variant product to a simple one via a unitary action
+When you are on a product page, at the variant level, you can decide to convert it to a simple product very easily.
+1. Click on `...`
+1. Then `Convert to a simple product`
+1. Confirm
+
+Your variant product becomes a simple product but don't worry, all of its former values, categories and associations will be saved.
```

Comment:
I would move this line above, to explain that no matter the way to convert (UI, import, API, ...) the categories, associations and values are kept

Reply:
you're right !! thanks :)